extractor/src/main/java/org/schabi/newpipe/extractor/utils/Utils.java

```diff
@@ -27,6 +33,24 @@ public static String removeNonDigitCharacters(String toRemove) {
         return toRemove.replaceAll("\\D+", "");
     }
 
+    /**
+     * <p>Remove a number from a string.</p>
+     * <p>Examples:</p>
+     * <ul>
+     *     <li>"123" -&gt; "123"</li>
```

Comment:
I guess this is a typo and you mean
```suggestion
     *     <li>"123" -&gt; ""</li>
```

Reply:
thank you